packages/integration-stable/setupTeardown.ts

```diff
@@ -15,7 +15,7 @@ export async function setupTests() {
 }
 
 async function cleanupData(client: Client) {
-  await client.execute(`reset schema to initial`);
+  await client.execute(`delete PgVectorTest`);
```

Comment:
Whoops! I erroneously thought `reset schema to initial` just deleted all the user defined objects, but it actually... resets the schema to the initial state before any migrations were run. 🤦 

Still need to figure out a query that will delete all user-defined types!